crates/ark/src/sys/windows/interface.rs

```diff
@@ -67,6 +67,20 @@ pub fn setup_r(mut _args: Vec<*mut c_char>) {
         // R_DefParamsEx(params, bindings::RSTART_VERSION as i32);
         R_DefParamsEx(params, 0);
 
+        let args_str: Vec<String> = args
+            .iter()
+            .map(|&arg| {
+                let cstr = CStr::from_ptr(arg);
+                cstr.to_string_lossy().to_string()
+            })
+            .collect();
+
+        if args_str.contains(&"--save".to_string()) {
```

Comment:
Should also handle `--no-save`.

I think that suggests you need to loop over the arguments and handle them one by one (last one wins) as in https://github.com/r-devel/r-svn/blob/48256015cc3c3e7cc2235179adccc60a641f6603/src/main/CommandLineArgs.c#L102.

(That is, if we're not going to rely on `cmdlineoptions()` to set these)